base/src/BrightnessContrastControl.cpp

```diff
@@ -153,7 +159,7 @@ BrightnessContrastControlProps BrightnessContrastControl::getProps()
 
 bool BrightnessContrastControl::handlePropsChange(frame_sp &frame)
```

Comment:
referer #64 for the correct name for the module. It should be BrightnessContrastXform. I udnerstand that this was created before we figired out the conventions, please rename the class in this go

Reply:
I have made the requested changes , please review them